models/models.js

```diff
@@ -28,32 +28,75 @@ function selectArticle(id) {
     db.query(`SELECT * FROM articles WHERE article_id = $1;`, [id])
       .then((result) => {
         if (result.rows.length === 0) {
-          return Promise.reject({ status: 404, msg: "article does not exist" });
+          throw {
+            status: 404,
+            msg: "article does not exist",
+          };
         }
-        resolve(result.rows[0]);
+        return db.query(
+          `ALTER TABLE articles ADD COLUMN IF NOT EXISTS comment_count INT DEFAULT 0;`
```

Comment:
We should not add a column but join the two tables to display the information we want